framework/src/executioners/EigenExecutionerBase.C

```diff
@@ -319,7 +319,7 @@ EigenExecutionerBase::inversePowerIteration(unsigned int min_iter,
 
     if (echo)
       _console << " ________________________________________________________________________________ "
-               << std::endl;
+               << std::endl << std::flush;
```

Comment:
@permcody you can checkout my yak branch `coupled_reactor`. The input is under `yak/tests/actions/neutron_diffusion_cfem/master.i`. We can use it to fix all minor output issues.